product_docs/docs/epas/16/upgrading/minor_update/performing_a_minor_version_update_of_Deb_installation.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Performing a minor version update of a Debian installation"
+description: "For Debian package installs, how to perform a minor version upgrade to a more recent version."
+---
+
+If you are upgrading a Debian-based system (or a compatible system such as Ubuntu), then you used a Debian package to install EDB Postgres Advanced Server. To perform a minor upgrade on these systems, use the `apt-get` package manager.
```

Comment:
Rest of the doc says update, I think we can have either upgrade or update across the doc

Reply:
Hey @nidhibhammar! 
Originally I wanted to call major procedures _upgrades_ and minor procedures _updates_ but I realize I wasn't consistent and thinking about it more, the customer will rather look for "upgrades" rather than "updates" so I am going to rephrase.